tests/phpunit/api/v3/ReportTemplateTest.php

```diff
@@ -139,6 +139,19 @@ public function testReportTemplateGetRowsContactSummary() {
      */
   }
 
+  /**
+   * Test getrows on Mailing Opened report.
+   */
+  public function testReportTemplateGetRowsMailingUniqueOpened() {
+    $description = "Retrieve rows from a mailing opened report template.";
+    $result = $this->callAPIAndDocument('report_template', 'getrows', array(
```

Comment:
Just use callApiSuccess unless you are clear that you want to create an example (& if you do that you should check it in)